internal/server/warning.go

```diff
@@ -36,17 +38,33 @@ func Warning(client WarningClient, tmpl template.Template) Handler {
 			return err
 		}
 
+		donor, err := client.Person(ctx, personId)
```

Comment:
Does this mean a donor is now always required to be able to create a warning? Are there any situations where we create a warning without a donor? (I guess a warning is always on a case, and a case always has a donor, though?) (Just for my understanding.)

Reply:
Up until now, warnings have always been created on a donor (by adding the url param ID to the POST request, except now i'm just fetching the donor for extra case data here). Yeah you've got it, a donor can exist without a case, but a case can't exist without a donor